src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -116,6 +116,10 @@
             {collection-image :image-url
              collection-name  :name}    collection-data
             {collectible-name :name}    collectible-data]
+        (rn/use-effect
+         (fn []
+           #(rf/dispatch [:wallet/clear-last-collectible-details]))
+         [])
```

Comment:
I think this empty vector can be omitted

Reply:
Nice, I didn't know about this. Done

Reply:
I want to run the code only in componentWillUnmount